### PR TITLE
CY-1736 - Updated UI developers documentation (cont.)

### PR DESCRIPTION
**Pull request by qooban**
Added package description to README.md file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,21 +51,54 @@ Changes in the source code shall be hot loaded to the development version of the
...
-## Deployment
-You can create a package and deploy it on a remote Cloudify Manager server.
+## Package
+
+You can create application package and deploy it on a remote Cloudify Manager server.
+
+### Package creation
+
+To create application package:  
+1. Create production build by running: `npm run build`
```
Comment (kubama) [inline README.md]: No full stop character at the end of this point (the two points below are ended with full stop)

File: README.md [hunk 2]
```diff
@@ -51,21 +51,54 @@ Changes in the source code shall be hot loaded to the development version of the
...
+1. Pack all necessary files into archive by running: `npm run zip`. 
+1. Application package will be in `stage.tar.gz` file in repository main directory. 
+
+### Package upload
+
+To upload the package to the remote Cloudify Manager:
+1. Define path to private SSH key to access Cloudify Manager: `export SSH_KEY_PATH=<PATH>`
+1. Define Cloudify Manager IP adress: `export MANAGER_IP=<MANAGER_IP>`
+1. Upload package to the Cloudify Manager and restart UI services: `npm run upload`
+1. Open browser and go to page `http://<MANAGER_IP>` to see if application is running.
```
Comment (kubama) [inline README.md]: Full stop at the end of the line - lines above are not ending with full stop.

File: README.md [hunk 3]
```diff
@@ -51,21 +51,54 @@ Changes in the source code shall be hot loaded to the development version of the
...
+Package archive contains the following resources:
+
+* `backend` - Stage Backend - whole [backend](./backend) folder
+* `conf` - configuration files (see [Configuration](./conf/README.md) for details)
+* `dist` - Stage Frontend - directory created by Webpack according to the [production configuration](./webpack.config-prod.js)
+  * `appData` - built-in application data
+    * `tours` - built-in tours - whole [tours](./tours) folder
+    * `widgets` - built-in widgets
+      * `<widget-name>` - every widget has its own folder 
+        * `widget.js` - minified widget bundle file (+ brotli-compressed widget and gzip-compressed widget bundles)
```
Comment (kubama) [inline README.md]: This is confusing to me - widget.js contains brotli and gzip compressed bundles?

File: README.md [hunk 4]
```diff
@@ -51,21 +51,54 @@ Changes in the source code shall be hot loaded to the development version of the
...
+        * `widget.png` - widget thumbnail
+      * ...
+    * `templates` - built-in templates - whole [templates](./templates) folder
+      * `pages` - built-in pages - whole [templates/pages](./templates/pages) folder 
+  * `static` - static files
+    * `images` - image files
+    * `fonts` - font files
+    * `js` - JavaScript bundles
+      * `main.bundle.js` - bundle created from client-side application source code (+ brotli-compressed and gzip-compressed bundle)
+      * `vendor.bundle.js` - bundle created from client-side application external depenencies (+ brotli-compressed and gzip-compressed bundle)
```
Comment (kubama) [inline README.md]: Same kind of confusion as above

File: README.md (hunk 3 above)
Comment (qooban) [reply README.md]: Fixed.

File: README.md (hunk 4 above)
Comment (qooban) [reply README.md]: Fixed.

File: README.md (hunk 2 above)
Comment (qooban) [reply README.md]: Fixed.

File: README.md (hunk 1 above)
Comment (qooban) [reply README.md]: Fixed.

**Pull request merged**